### PR TITLE
Delayed job failures for the editor

### PR DESCRIPTION
**Pull request by H3ll3m4**
Adding a walkthrough for the delayed job failures for the editor

### CODE REVIEW

File: source/documentation/runbooks/delayed-job-failures-editor.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+last_reviewed_on: 2023-01-30
+review_in: 3 months
+---
+
+
+<% if current_page.url != "/" %># <%= current_page.data.title %><% end %>
+
+It happens the job could not be processed because the corresponding PublishService id wasn't found.
+
+1. First identify the environment, as rule of thumb what is in the alert channel is live environment while alerts in the loaw severity channel affect test environment.
```
Comment (StevenLeighton21) [inline source/documentation/runbooks/delayed-job-failures-editor.html.md.erb:13]:
```suggestion
1. First identify the environment, as rule of thumb what is in the alert channel is live environment while alerts in the low severity channel affect test environment.
```

**Review by StevenLeighton21 — APPROVED** [review]
What does it mean if there is an entry in the db for the published services with that id? Or is that just a never going to happen thing and this is a runbook for cleaning up mismatched delayed jobs?

File: source/documentation/runbooks/delayed-job-failures-editor.html.md.erb (hunk 1 above)
Comment (H3ll3m4) [reply source/documentation/runbooks/delayed-job-failures-editor.html.md.erb]: thank you

**Comment by H3ll3m4** [comment]
> What does it mean if there is an entry in the db for the published services with that id? Or is that just a never going to happen thing and this is a runbook for cleaning up mismatched delayed jobs?

This runbook is for this situation that I encounter twice being in support last week. We could investigate why is that happening and fix it if that happens too often.

File: source/documentation/runbooks/delayed-job-failures-editor.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+title: Editor's Delayed Job Failures
+category: runbooks
+last_reviewed_on: 2023-01-30
+review_in: 3 months
+---
+
+
+<% if current_page.url != "/" %># <%= current_page.data.title %><% end %>
+
+It happens the job could not be processed because the corresponding PublishService id wasn't found.
```
Comment (StevenLeighton21) [inline source/documentation/runbooks/delayed-job-failures-editor.html.md.erb:11]: Suggest adding a line something like 'in this case there is no issue, we just need to clean up the failed Delayed Job', based on https://github.com/ministryofjustice/moj-forms-tech-docs/pull/44#issuecomment-1410088233

**Review by StevenLeighton21 — APPROVED** [review]

**Pull request merged**